Pipfile

```diff
@@ -27,13 +27,15 @@ troposphere = "==3.1.1"
 Unidecode = "==1.2.*"
 
 [dev-packages]
-autopep8 = "~=1.3"
-pyflakes = "~=2.1"
-pylint = "~=2.0"
-pytest = "~=7.2.0"
-pytest-cov = "==2.5.*"
-pytest-forked = "~=0.2"
-coverage = "~=4.5"
+autopep8 = "~=2.0"
+pyflakes = "~=3.0"
+pylint = "~=2.17"
+pytest = "~=7.3"
+pytest-cov = "~=4.1"
+# lsh@2023-06-14: not sure why we needed this one.
+# lets see how we do without it.
+#pytest-forked = "~=1.6"
```

Comment:
looks like it was introduced here:
https://github.com/elifesciences/builder/commit/5e902cb8c5ad665a6b96652d182a3d23a8ea2737